### PR TITLE
[MOV] ecommerce: moving product management page

### PR DESCRIPTION
**Pull request by Donapi**
[task](https://www.odoo.com/odoo/project/3835/tasks/4056328?cids=1)

### CODE REVIEW

File: content/applications/websites/ecommerce/products.rst [hunk 1]
```diff
@@ -1,3 +1,9 @@
+:show-content:
+
+========
+Products
+========
+
 ==================
 Product management
 ==================
```
Comment (auva-odoo) [inline content/applications/websites/ecommerce/products.rst:9]: Did you mean to remove this? There are currently 2 h1 in this rst ...

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hi @Donapi! I noticed two small issues:
- There's a double h1 (see my comment in the review).
- It looks like the redirect from managing_products to products might have been missed.
I also suggest changing the PR name slightly to include the page you moved. Since you'll be moving a lot of pages while restructuring the commerce doc, this will help keep things organized better.
Thanks!

File: content/applications/websites/ecommerce/products.rst (hunk 1 above)
Comment (ksc-odoo) [inline content/applications/websites/ecommerce/products.rst:9]:
```suggestion
Product management
==================
```
I agree with @auva-odoo ... this is how this header should be formatted in this case

**Review by ksc-odoo — APPROVED** [review]
Apart from the one comment I left (which supported @auva-odoo 's original comment), this one looks ready to go. Approving now. 👍

File: redirects/17.0.txt [hunk 2]
```diff
@@ -75,3 +75,7 @@ applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/p
 
 applications/services/helpdesk/overview/getting_started.rst applications/services/helpdesk.rst                                                            #helpdesk/overview/getting_started --> helpdesk
 applications/services/project/tasks/email_alias.rst applications/services/project/tasks/task_creation.rst                                                 #project/tasks/email_alias --> task_creation
+
+# applications/website
+
+applications/websites/ecommerce/managing_products/products.rst applications/websites/ecommerce/products.rst
```
Comment (auva-odoo) [inline redirects/17.0.txt:81]: Sorry if I wasn't clear earlier: You need to add one for each of the sub-pages as well (catalog, variants, etc.)

**Review by auva-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**